### DISCUSSION TITLE
Update local beatmap files to latest

### DISCUSSION THREAD

**Discussion by mzhang28** [ppy/osu#14445] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/1700346/130361069-450ff67f-0562-47ea-a7ce-76fd4e53d6f4.png)

Can't find the equivalent of this button in lazer right now (2021.820.0 build), and since you can't re-download maps you already have, updating requires first deleting the map, then searching the map up again through the download panel, then downloading.

1. Is this planned to be added back to lazer? I don't see a pending issue about it
2. Can this update button also pull beatmap-related assets? (for example, in stable, it's not possible to pull a new mp3 via that button, which means that if the offset is updated, players will have to re-download the map since the map's new offset will no longer correspond to the old mp3.) (also not possible to pull maps with new hitsounds or storyboard elements without re-downloading)

Also do these discussions make it into the issue tracker at all or should I open an issue instead...

**Comment by LittleEndu** [comment, marked as answer]
> Also do these discussions make it into the issue tracker at all or should I open an issue instead.

They do (there's that "Create issue from discussion" button), but most of the features that are present in stable but haven't been implemented in lazer yet don't get their own issue opened because the team is well aware that they need to be implemented.

**Reply by mzhang28** [reply]
Ah didn't see the button. I'll still open one since the feature isn't there currently and I would like to know when and discuss how it will be reimplemented.

**Reply by LittleEndu** [reply]
That's what these discussions are for, yeah. I think the team preferers discussions to be opened over issues for exactly discussing the features first.

**Reply by rednir** [reply]
> Ah didn't see the button. I'll still open one since the feature isn't there currently and I would like to know when and discuss how it will be reimplemented.

https://github.com/ppy/osu/issues/4823 is this not the same? @iptq

**Reply by bdach** [reply]
@rednir that is indeed the same issue.

**Reply by mzhang28** [reply]
holy shhit my bad i literally went thru all of these issues and did not see this one
